src/Illuminate/Database/Console/ShowModelCommand.php

```diff
@@ -225,6 +226,25 @@ protected function getRelations($model)
             ->values();
     }
 
+    /**
+     * Get the Events that the model dispatches.
+     *
+     * @param  \Illuminate\Database\Eloquent\Model  $model
+     * @return \Illuminate\Support\Collection
+     */
+    protected function getEvents($model)
+    {
+        $formatted = [];
+        foreach ($model->dispatchesEvents() as $event => $class) {
+            $formatted[] = [
+                'event' => $event,
+                'class' => $class,
+            ];
+        }
+
+        return collect($formatted);
```

Comment:
You could probably get rid of that temp variable since you're returning a collection:

```php
return collect($file->dispatchesEvents())
    ->flatMap(fn ($class, $event) => [
        'event' => $event,
        'class' => $class
    ]);
```

Reply:
That's true, thanks for the heads up, I'll update it!

Reply:
@mateusjatenee I used a normal map instead of the flatMap but I updated it xD